src/main/resources/XWiki/Macros/TeamConfigurationClass.xml

```diff
@@ -0,0 +1,134 @@
+      <customDisplay/>
+      <defaultValue>0</defaultValue>
+      <disabled>0</disabled>
+      <displayFormType>select</displayFormType>
+      <displayType/>
+      <hint/>
+      <name>disableTools</name>
+      <number>3</number>
+      <prettyName>Disable tools letting filter and show usernames</prettyName>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+    </disableTools>
+    <letterAvatarBgColor>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <hint>#0A6</hint>
+      <name>letterAvatarBgColor</name>
+      <number>6</number>
+      <picker>0</picker>
+      <prettyName>Background color for letter avatars</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+    </letterAvatarBgColor>
+    <letterAvatarFontColor>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <hint>white</hint>
+      <name>letterAvatarFontColor</name>
+      <number>7</number>
+      <picker>0</picker>
+      <prettyName>Font color for the letter avatars</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+    </letterAvatarFontColor>
+    <requireExternalAuth>
+      <customDisplay/>
+      <defaultValue>0</defaultValue>
+      <disabled>0</disabled>
+      <displayFormType>select</displayFormType>
+      <displayType/>
+      <hint/>
+      <name>requireExternalAuth</name>
+      <number>2</number>
+      <prettyName>Require that users are registered in LDAP or with OpenIDC</prettyName>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+    </requireExternalAuth>
+    <showUsernames>
+      <customDisplay/>
+      <defaultValue>0</defaultValue>
+      <disabled>0</disabled>
+      <displayFormType>select</displayFormType>
+      <displayType/>
+      <hint/>
+      <name>showUsernames</name>
+      <number>1</number>
+      <prettyName>Show usernames under their avatars</prettyName>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+    </showUsernames>
+    <size>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <hint/>
+      <name>size</name>
+      <number>4</number>
+      <numberType>integer</numberType>
+      <prettyName>Size of the avatars, in pixels</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+    </size>
+  </class>
```

Comment:
Missing default class sheet binding (to show the list of properties when accessing the class page).